CHANGES.md

```diff
@@ -268,6 +268,7 @@ _This is an npm-only release to fix a publishing issue_.
 * Added `computeLineSegmentLineSegmentIntersection` to `Intersections2D`. [#7228](https://github.com/AnalyticalGraphicsInc/Cesium/pull/7228)
 * Added ability to load availability progressively from a quantized mesh extension instead of upfront. This will speed up load time and reduce memory usage. [#7196](https://github.com/AnalyticalGraphicsInc/cesium/pull/7196)
 * Added the ability to apply styles to 3D Tilesets that don't contain features. [#7255](https://github.com/AnalyticalGraphicsInc/Cesium/pull/7255)
+* Added support for emojis in Labels.
```

Comment:
Looks like this got merged with "support for emojis", but more importantly, the CHANGES message is back-dated to Cesium 1.52 from last December.

I've often wondered if Cesium needs an automated process to detect and alert for changes to CHANGES for released versions.

Reply:
Nevermind, Matt fixed it in e0e8062907d.

Reply:
Sorry, meant to leave a comment on this PR linking to that.  Thanks @emackey!